chainnotifier_client.go

```diff
@@ -0,0 +1,239 @@
+package lndclient
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightninglabs/loop/swap"
```

Comment:
Can this import be removed? Or do you want to do this in a follow-up PR so this is a pure code move?